src/Domain/Creators/Cylinder.hpp

```diff
@@ -20,8 +20,7 @@
 /// \cond
 namespace domain {
 namespace CoordinateMaps {
-class Affine;
-class Equiangular;
+class Interval;
```

Comment:
I'm going through the commits again, checking the diffs. Is this correct by the way?